sklearn/metrics/_ranking.py

```diff
@@ -944,10 +944,16 @@ def precision_recall_curve(
     precision : ndarray of shape (n_thresholds + 1,)
         Precision values such that element i is the precision of
         predictions with score >= thresholds[i] and the last element is 1.
+        Note that this means that the positive class must be positively
+        correlated with score. See examples for a case where the negative
+        class is positively correlated with score.
```

Comment:
I think this can go in the main docstring, maybe at the end ~line 904? 

I think it's important information and the user shouldn't need to scroll to the 'Returns' section to see it.

Reply:
I added it to the "Binary classification " chapter as this is where things are defined. Do you think this makes sense?